src/database/migrations/20221101124519_create_prices_table.ts

```diff
@@ -0,0 +1,13 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.createTable('prices', (table) => {
```

Comment:
Tähän tauluun varmaan tehdään kyselyitä tyylillä `WHERE gas=<kaasu> AND created_at=(select MAX(created_at) FROM prices)` ??

Reply:
Entiä menikö syntaksi oikein :D

Reply:
Voi varmaan jotenkin noinkin, itse aattelin tyylillä:
`SELECT price FROM prices WHERE gas=<kaasu> ORDER BY created_at DESC LIMIT 1;

Reply:
Tiedä häntä, en ole sql-jumala

Reply:
Sanoisin että tommoinen subquery on hitaampi. Ainakin niitä pyritään välttämään jos mahdollista. Anyways dataa ei ole niin paljon etteikö `ORDER BY` toimisi ihan yhtä nopeasti. Ja se on selkeämpi

Reply:
Niin en kyl aatellu tuota subquery twistiä vaan perustin ajatukseni siihen että järjestäminen on varmaankin nlogn operaatio ja suurimman arvon etsiminen lineaarinen mutta ei nyt ruveta harjoittamaan ennenaikaista optimointia

Reply:
_We should think about small efficiencies, say about 97% of the time: premature optimization is the root of all joy._